src/styles/optionRowStyles/types.ts

```diff
@@ -1,6 +1,5 @@
-import {CSSProperties} from 'react';
 import {ViewStyle} from 'react-native';
 
-type OptionRowStyles = CSSProperties | ViewStyle;
+type CompactContentContainerStyles = ViewStyle;
```

Comment:
why is there a need to rename this?

Reply:
Probably it was a leftover, let's remove `CompactContentContainerStyles` and use `ViewStyle` instead

Reply:
In that case I will need to remove `types.ts` and this is module where there are env specific file, do we want to do that? CC: @blazejkustra @hayata-suenaga

Reply:
let's do that in a follow up PR if necessary